test-framework/sudo-compliance-tests/src/sudo/flag_preserve_environment.rs

```diff
@@ -0,0 +1,63 @@
+use sudo_test::{Command, Env};
```

Comment:
Maybe also have a test for using `-E` and explicit env vars with disjoint names to check that `-E` doesn't clear the list of explicit env vars?

Reply:
Something like b20da66 ?